spring-data-examples/eviction/README.md

```diff
@@ -0,0 +1,23 @@
+# Eviction Example
+
+In this com we will show you ways to automatically evict data from your region if certain entry count or memory thresholds are reached.
```

Comment:
What is a `com`?

Reply:
Darn, I thought I'd fixed all the places it did that. When I renamed the example package IJ changed a bunch of random references to the word "example". I'll clean it up.

Reply:
Should be fixed now